### PR TITLE
Devise set up

### PR DESCRIPTION
**Pull request by emestabillo**
Updated

### CODE REVIEW

File: config/environments/production.rb [hunk 1]
```diff
@@ -1,4 +1,5 @@
 Rails.application.configure do
+  config.action_mailer.default_url_options = { host: 'chess-app-knightster.herokuapp.com' }
```
Comment (jacobwgillespie) [inline config/environments/production.rb]: I just set up the Heroku app - can you update this to be `knightster.herokuapp.com`?

**Comment by jacobwgillespie** [comment]
Hey @emestabillo this was closed without being merged - was that intentional?

**Comment by emestabillo** [comment]
Sorry! I tried to work on it through my phone but I must've pressed another button. Will undo it. Thanks Jacob!

**Comment by jacobwgillespie** [comment]
No worries!  Just wanted to make sure your code actually got in there 😄

**Pull request merged**